router/wrap_unwrap.gno

```diff
@@ -11,11 +11,17 @@ import (
 
 func wrap(ugnotAmount uint64) {
 	if ugnotAmount <= 0 {
-		panic(ufmt.Sprintf("cannot wrap 0 ugnot"))
+		panic(addDetailToError(
+			errWrapUnwrap,
```

Comment:
The function and error handling of unwrap seems to be different.
When unwrap, no error is raised when the quantity is zero.

Reply:
`ugnotAmount` for `func wrap()` is amount of native ugnot send to wrap.
> without senting coin, we can't unwrap it.

`wugnotAmount` for `func unwrap()` is amount of grc20 wugnot to unwrap.
> with 0 amount of wugnot, we can just skip unwrap.